### PR TITLE
doc: added macos shortcuts

### PR DESCRIPTION
**Pull request by djdjz7**

### CODE REVIEW

**Review by 000lbh — DISMISSED** [review]

File: basic/07-drive-your-computer-3.md [hunk 1]
```diff
@@ -112,7 +112,39 @@ Windows 下打开任务管理器检查有无不必要启动项，加以禁止即
...
+  - `Command+M`: 最小化当前窗口
+  - `Command+Space`: 打开聚焦搜索
+  - `Option-Command-Esc`: 打开 `强制退出应用程序` 窗口，类似于任务管理器
+  - `Control+左右方向键`: 切换虚拟桌面
+
+  ::: tip
+  事实上，上述快捷键不一定是绝对的，例如:
+
+  - 在多标签页的程序（VS Code、浏览器等）中，`Command+W` 用于关闭当前标签页，当没有标签页可以关闭时，关闭窗口。
+  - 一般情况下，撤销对应的快捷键是 `Command+Shift+Z`，但是在某些第三方程序，例如 Microsoft Word 中，`Command+Z` 也可以用于撤销。
```
Comment (leavelet) [inline basic/07-drive-your-computer-3.md:141]: 这里确定没错吗，一般情况下撤销就是 command+z 吧？

**Review by leavelet — CHANGES_REQUESTED** [review]
还可以提一下符号与文字的对应，比如⌘对应command之类的

**Comment by djdjz7** [comment]
> 还可以提一下符号与文字的对应，比如⌘对应command之类的

这几个字符跨平台显示效果不太好，我准备画几个svg上去试一试

File: basic/07-drive-your-computer-3.md (hunk 1 above)
Comment (djdjz7) [reply basic/07-drive-your-computer-3.md:141]: 嘶写错了抱歉，这里是重做有问题

**Comment by HolgerHuo** [comment]
Vue part looks good to me. But maybe we should consider make it global or consider some other ways of implementing this feature, e.g., plain svg files with global css, or special unicode font, so as to enable other features, like pdf version of this lecture, publishing to other platforms.

File: basic/07-drive-your-computer-3.md [hunk 2]
```diff
@@ -112,7 +112,48 @@ Windows 下打开任务管理器检查有无不必要启动项，加以禁止即
...
+  | Command | <MacOSCommandKey /> |
+  | Option  | <MacOSOptionKey />  |
+  | Shift   |  <MacOSShiftKey />  |
+  | Control | <MacOSControlKey /> |
+
+  ::: tip
+  事实上，上述快捷键不一定是绝对的，例如:
+
+  - 在多标签页的程序（VS Code、浏览器等）中，`Command+W` 用于关闭当前标签页，当没有标签页可以关闭时，关闭窗口。
+  - 一般情况下，重做对应的快捷键是 `Command+Shift+Z`，但是在某些第三方程序，例如 Microsoft Word 中，`Command+Y` 也可以用于重做。
```
Comment (HolgerHuo) [inline basic/07-drive-your-computer-3.md:150]: This line is unfixed yet.

**Comment by HolgerHuo** [comment]
I'm unfamiliar with macos shortcuts part. I'll approve now for vue part, macos part should be reviewed otherwise.

**Review by HolgerHuo — APPROVED** [review]

**Review by 000lbh — APPROVED** [review]

**Review by leavelet — APPROVED** [review]

**Pull request merged**